sympy/series/limits.py

```diff
@@ -135,7 +136,12 @@ def doit(self, **hints):
 
         if not e.has(z):
             return e
-
+        #limit is not working with binomial,gruntz raise an exception
```

Comment:
> ready now to get merged

No yet. Please add spaces after the `#` and format your commit message with a blank line after the title. Please see the reference/link that I referred to if you have questions.